DependencyInjection/GraphQLiteCompilerPass.php

```diff
@@ -40,7 +41,6 @@
 use TheCodingMachine\CacheUtils\ClassBoundCacheContractInterface;
 use TheCodingMachine\CacheUtils\ClassBoundMemoryAdapter;
 use TheCodingMachine\CacheUtils\FileBoundCache;
-use TheCodingMachine\ClassExplorer\Glob\GlobClassExplorer;
```

Comment:
I cannot find where we are removing this dependency, it looks like it was not explicitly required.

Reply:
class explorer was a lib used by graphqlite but we dropped it in version 7

Reply:
[here is the PR](https://github.com/thecodingmachine/graphqlite/pull/664)